clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -52,19 +52,26 @@ class checker_no_abort_t : public invariant_checker_t {
     {
     }
     std::vector<std::string> errors;
+    std::vector<memref_tid_t> error_tids;
+    std::vector<uint64_t> error_refs;
 
```

Comment:
Does the view tool currently print at regular intervals the count of memrefs already printed? If not, will be good to add it; it'll make finding the memrefs from `error_refs` easiers.

Reply:
No.  Probably it would be a per-line prefix, like line numbers in less -- not sure it would need an option to turn off since the numbers can get quite large.  This would be a separate PR.